README.md

```diff
@@ -3,3 +3,7 @@
 This repository will eventually house the source material for the documentation for the [OpenTracing](http://opentracing.io/documentation/) project.
 
 For the time being, that source material can be found in [this repository](https://github.com/opentracing/opentracing.io/tree/master/_docs).
+
+## License
+
+By contributing to OpenTracing documentation repository, you agree that your contributions will be licensed under its Apache 2.0 License.
```

Comment:
You can add a link by specifying the link text within `[]` and the URL within `()`. As the `LICENSE` file is at the same directory as the readme file, the URL can be relative. Like this:

```
... will be licensed under its [Apache 2.0 License](./LICENSE).
```